modules/bullet/SCsub

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python
+
+Import('env')
+
+# build only version 2
+# Bullet 2.87
+bullet_src__2_x = [
+        # BulletCollision
+          "BulletCollision/BroadphaseCollision/btAxisSweep3.cpp"
+        , "BulletCollision/BroadphaseCollision/btBroadphaseProxy.cpp"
```

Comment:
It's a matter of style, but I think it's best to be consistent with other moduels and put the comma after the string, not before.